src/main/kotlin/pt/up/fe/ni/website/backend/controller/ErrorController.kt

```diff
@@ -0,0 +1,13 @@
+package pt.up.fe.ni.website.backend.controller
+
+import org.springframework.boot.web.servlet.error.ErrorController
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ErrorController : ErrorController {
+    val errorKey : String = "error"
+
+    @RequestMapping("/**")
+    fun endpointNotFound(): Map<String, String> = mapOf(errorKey to "invalid endpoint")
+}
```

Comment:
No empty line (I guess the linter would say something about this)

Reply:
Indeed the linter (now merged from develop) warns about this. Fixed